static-assets/scripts/host.js

```diff
@@ -581,7 +581,6 @@
 
     path = path.replace('//', '/');
 
-    CStudioAuthoring.ContextualNav.WcmActiveContent?.disableNav(true);
```

Comment:
I think these may be covering other scenarios and we should leave them as long as they cause no trouble.